resources/views/cities/index.blade.php

```diff
@@ -0,0 +1,167 @@
+                            <x-table-button onclick="deleteCity(' + item.id + ')">Delete</x-table-button>\
+                            </x-table-data>\
+                            </tr>'
+                    );
+                });
+
+                // Update pagination links
+                $('#pagination').html('');
+                if (response.pagination) {
+                    $('#pagination').append(
+                        '<nav class="flex items-center justify-between border-t border-gray-200 px-4 py-3 sm:px-6"\
+                                aria-label="Pagination">\
+                                <div class="hidden sm:block">\
+                                    <p class="text-sm text-gray-700">Showing page<span class="font-medium"> ' + response.pagination.currentPage + ' \
+                                        </span>of<span class="font-medium"> ' + response.pagination.totalPages + ' \
+                                        </span></p>\
+                                </div>\
+                                <div id="paginationBtn" class="flex flex-1 justify-between sm:justify-end">\
+                                </div>\
+                            </nav>'
+                    );
+                    if (response.pagination.currentPage > 1) {
+                        $('#paginationBtn').append(
+                            '<a onclick="fetchCities(' + (response.pagination.currentPage - 1) +
+                            ')"\
+                                class="relative inline-flex items-center rounded-md px-3 py-2 text-sm font-semibold text-gray-900 ring-1 ring-inset ring-gray-300 hover:bg-gray-200 focus-visible:outline-offset-0">Previous</a>'
+                        );
+                    } else {
+                        $('#paginationBtn').append(
+                            '<a class="relative ml-3 inline-flex items-center rounded-md px-3 py-2 text-sm font-semibold text-gray-400 ring-1 ring-inset ring-gray-200 focus-visible:outline-offset-0 cursor-not-allowed">Previous</a>'
+                        );
+                    }
+                    if (response.pagination.currentPage < response.pagination.totalPages) {
+                        $('#paginationBtn').append(
+                            '<a onclick="fetchCities(' + (response.pagination.currentPage + 1) +
+                            ')"\
+                                class="relative ml-3 inline-flex items-center rounded-md px-3 py-2 text-sm font-semibold text-gray-900 ring-1 ring-inset ring-gray-300 hover:bg-gray-200 focus-visible:outline-offset-0">Next</a>'
+                        );
+                    } else {
+                        $('#paginationBtn').append(
+                            '<a class="relative ml-3 inline-flex items-center rounded-md px-3 py-2 text-sm font-semibold text-gray-400 ring-1 ring-inset ring-gray-200 focus-visible:outline-offset-0 cursor-not-allowed">Next</a>'
+                        );
```

Comment:
What happens if you have an other paginated response in the future? You'd have to add all this logic again? Try something more reusable, check laravel documentation on displaying pagination here: https://laravel.com/docs/11.x/pagination#displaying-pagination-results
and also on customising the display editing the vendor here: https://laravel.com/docs/11.x/pagination#customizing-the-pagination-view